src/System.Runtime.InteropServices/ref/System.Runtime.InteropServices.cs

```diff
@@ -481,11 +511,18 @@ public partial class InvalidOleVariantTypeException : System.SystemException
         public InvalidOleVariantTypeException(string message) { }
         public InvalidOleVariantTypeException(string message, System.Exception inner) { }
     }
-    [AttributeUsage(AttributeTargets.Method, Inherited = false)]
-    public sealed class LCIDConversionAttribute : Attribute
+    [System.AttributeUsageAttribute(System.AttributeTargets.Method, Inherited=false)]
+    public sealed partial class LCIDConversionAttribute : System.Attribute
     {
         public LCIDConversionAttribute(int lcid) { }
-        public int Value { get; }
+        public int Value { get { throw null; } }
+    }
+    [System.AttributeUsageAttribute(System.AttributeTargets.Method, Inherited=false, AllowMultiple=false)]
+    public sealed partial class ManagedToNativeComInteropStubAttribute : System.Attribute
```

Comment:
I don't know why this is being exposed right now.

Reply:
This was already exposed as part of .Net Core 2.0. This PR isn't adding it, just normalizing the sort order/etc. https://github.com/dotnet/corefx/blob/bca44089343cf14cffd3cb0982b7e209692f6731/src/System.Runtime.InteropServices/ref/System.Runtime.InteropServices.cs#L698-L704

https://apisof.net/catalog/System.Runtime.InteropServices.ManagedToNativeComInteropStubAttribute